test/controllers/managing.officer.corporate.controller.spec.ts

```diff
@@ -134,17 +135,52 @@ describe("MANAGING_OFFICER CORPORATE controller", () => {
       expect(resp.text).toContain(SHOW_INFORMATION_ON_PUBLIC_REGISTER);
     });
 
-    test(`renders the ${MANAGING_OFFICER_CORPORATE_PAGE} page without public register jurisdiction field`, async () => {
+    /**
+     * This test should ideally be working but will skip it for now until a fix is worked out
+     */
```

Comment:
We should fix this now and not leave this to later.

Reply:
Sure..

Reply:
> We should fix this now and not leave this to later.

Fixed now.

Reply:
Nice. Guessing this was always broken. Must have worked by accident when running the app or the issue just not spotted.